projects/kit/components/toggle/toggle-options.ts

```diff
@@ -0,0 +1,28 @@
+import {InjectionToken} from '@angular/core';
+import {TuiContextWithImplicit} from '@taiga-ui/cdk';
+import {TuiSizeL} from '@taiga-ui/core';
+import {PolymorpheusContent} from '@tinkoff/ng-polymorpheus';
+
+export interface ToggleOptions {
+    readonly icons: Readonly<{
```

Comment:
Please also include other parameters: `singleColor`, `showIcons` and `size`. We probably do not need to include `showLoader` because it's a value people toggle on and off dynamically.

Reply:
Done